### DISCUSSION TITLE
truncate not working

### DISCUSSION THREAD

**Discussion by foremtehan** [tailwindlabs/tailwindcss#11269] [Help] [answered]
I want to truncate text when a screen is smaller than the full text, i used `max-w-full truncate` but it does not working:

https://play.tailwindcss.com/87Yd9F3NW6

Resize the window and you'll see it overflaws

**Comment by wongjn** [comment, marked as answer]
You'd want to override the implicit `min-width: min-content` in some of the horizontal flex layouts such that the text within the `truncate` element can overflow (and not one of its parent elements): https://play.tailwindcss.com/pmVQ4ObGvB

**Reply by foremtehan** [reply]
Thanks but it still does not applied the `...` at the end of the text

**Reply by wongjn** [reply]
The ellipsis points seem to appear at the end of the text on my machine, it might just be difficult to distinguish due to the lack of contrast with the background color. Here's a screenshot of it working with the background color changed:
![image](https://github.com/tailwindlabs/tailwindcss/assets/11310624/3ec5f104-1f4c-4870-a7fa-287c65e86c9b)

**Reply by foremtehan** [reply]
Yeah my bad, Thank you very much btw.
